bundles/org.openhab.binding.astro/src/main/java/org/openhab/binding/astro/internal/handler/AstroThingHandler.java

```diff
@@ -272,8 +270,8 @@ private void linkedChannelChange(ChannelUID channelUID, int step) {
      */
     private boolean isPositionalChannelLinked() {
         for (Channel channel : getThing().getChannels()) {
-            if (ArrayUtils.contains(getPositionalChannelIds(), channel.getUID().getId())
-                    && isLinked(channel.getUID().getId())) {
+            String id = channel.getUID().getId();
+            if (Arrays.asList(getPositionalChannelIds()).contains(id) && isLinked(id)) {
```

Comment:
I'd prefer if you cached `Arrays.asList(getPositionalChannelIds())` in a local variable so that it doesn't create a new array during each iteration of the for loop.